packages/agoric-cli/src/commands/perf.js

```diff
@@ -53,7 +55,10 @@ export const makePerfCommand = logger => {
       const sharedOpts = perf.opts();
       logger.warn({ sharedOpts, opts });
       const payloadStr = fs.readFileSync(opts.executeOffer).toString();
-      const { offer } = JSON.parse(JSON.parse(payloadStr).body);
+      const innerBody = JSON.parse(payloadStr).body;
+      innerBody.startsWith('#') ||
+        Fail`expected smallcaps encoded body: ${innerBody}`;
```

Comment:
this isn't backward compatible. is that on purpose?
any reason not to do something more like `.replace(/^#/, '')`?

Reply:
+1 to backwards compatibility. but not not worth delaying merging this rn

Reply:
I shoulda done that. But if this one works, I'll fix it correctly tomorrow. Leaving this conversation unresolved.

Reply:
#7713